client/app/assets/less/ant.less

```diff
@@ -289,7 +300,7 @@
 }
 
 .ant-popover {
-  z-index: 1000; // make sure it doesn't cover drawer
+  z-index: 2000; // make sure it doesn't cover drawer
```

Comment:
Not `@zindex-popover`?

Reply:
Actually, this can be removed altogether?

Reply:
Well, I actually don't know where it's used (tried to find any place but failed). Can you please tell me how to check this?

Reply:
<img width="731" alt="Screen Shot 2019-05-14 at 7 29 48" src="https://user-images.githubusercontent.com/486954/57701898-2eecaa00-761a-11e9-98eb-51bc12db2978.png">

Above modal but under drawer (click the "?" icon in the popover to make sure)

Reply:
I was so close :sweat_smile: Will check in a moment. Thanks for pointing on this!

Reply:
@ranbena Fixed. I increased z-index only for help drawer - to place it above all other Antd components.